app/javascript/components/base/base-button-component.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <button
+    class="mx-2 my-2 h-10 font-bold py-2 px-6 rounded shadow-md"
+    :class="elements.color"
+    @click="clickButton"
+  >
+    {{ elements.placeholder }}
+  </button>
+</template>
+
+<script>
+export default {
+  props: {
+    elements: { type: Object, required: true },
+  },
+  
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing spaces not allowed                    no-trailing-spaces